zipkin-web/src/main/scala/com/twitter/zipkin/web/TraceSummary.scala

```diff
@@ -18,7 +18,6 @@ package com.twitter.zipkin.web
 
 import com.twitter.finagle.tracing.SpanId
 import com.twitter.zipkin.common.{Trace, Endpoint, Span, SpanTreeEntry}
-import com.twitter.zipkin.web.Util.getRootMostSpan
 
 case class SpanTimestamp(name: String, timestamp: Long, duration: Long) {
```

Comment:
I was thinking (but it's a separate/later issue) that we should consider sending trace summaries over the wire instead of complete traces, in the default/index/search results page. And only send the complete trace data when you're looking at a single trace.
Anyways, should we go that direction, the code is still there in git.